faceted_search/src/config.json

```diff
@@ -0,0 +1,3 @@
+{
+    "solr_url": "https://mars.cyverse.org/thing/select"
+}
```

Comment:
This file will be replaced by the Docker build (depending on which iSB destination it is going to), but have a default value in place so things work out of the box for development.